tests/Aspire.Azure.AI.OpenAI.Tests/ConformanceTests.cs

```diff
@@ -25,6 +25,9 @@ public class ConformanceTests : ConformanceTests<AzureOpenAIClient, AzureOpenAIS
 
     protected override bool SupportsKeyedRegistrations => true;
 
+    protected override bool SupportsNamedConfig => true;
```

Comment:
How do you feel about flipping this in the base? Basically the base says `virtual bool SupportsNamedConfig => true;` and then only the places that don't override and return false.

Reply:
Yeah, it made sense to have this false-by-default before I add support for everything but flipped it does make sense now. 😄